ops/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
```

Comment:
generally we wouldn't update the copyright of a file, if you do, then it should be a range, but otherwise the position is the "oldest time when the file was created"

Reply:
Done.